### PR TITLE
Add 'Energy and Infrastructure' checkboxes to Large capital profile

### PR DESCRIPTION
**Pull request by paulgain**
Added 15 **Energy and infrastructure** checkboxes to Investor requirements within Large capital profile. 

1.  Biofuel
2.  Biomass
3.  Direct heating
4.  Energy from waste
5.  Energy storage
6.  Gas fired power
7.  Nuclear
8.  Regulated assets
9.  Smart energy
10. Solar power
11. Transport
12. Wave and tidal
13. Windpower (offshore)
14. Windpower (onshore)
15.Upstream oil and gas

**Note: there is a select all option to follow**

Display what the user has selected within the profile read-only view.

https://uktrade.atlassian.net/browse/IPBETA-272

**Energy and infrastructure edit view**

assets-classes-interest-edit

**Energy and infrastructure read-only view**

assets-classes-interest-read-only

**Mobile**

mobile

**Checklist**

- [ ] Has the branch been rebased to develop?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/companies/apps/investments/large-capital-profile/styles.scss [hunk 1]
```diff
@@ -42,22 +44,56 @@ nav.govuk-tabs {
...
     }
-    .types-of-investment {
-      @include flexWrap;
-      height: 205px;
+    &.investor-requirements-edit  {
+      .restrictions,
+      .deal-ticket-sizes {
+        @include flexWrap;
+        width: 660px;
+        height: 152px;
```
Comment (yozzo) [inline src/apps/companies/apps/investments/large-capital-profile/styles.scss]: do these fixed numbers change if the user has a larger font setting?

Comment (paulgain) [reply src/apps/companies/apps/investments/large-capital-profile/styles.scss]: Are you suggesting percentages instead?

Comment (yozzo) [reply src/apps/companies/apps/investments/large-capital-profile/styles.scss]: em's or rem's are relative to font size and could work better for vertical dimensions

Comment (paulgain) [reply src/apps/companies/apps/investments/large-capital-profile/styles.scss]: Thanks @yozzo

**Pull request merged**